README.md

```diff
@@ -1,44 +1,38 @@
-| index | [example]     |
-|-------|---------------|
+|[README.md](#)| [Add-on template](https://github.com/mozilla/shield-studies-addon-template/)|[Engineering hints](./docs/engineering.md)|[More documentation](./docs/)|[Shield - Mozilla Wiki](https://wiki.mozilla.org/Firefox/Shield)
+|-------|---------------|----|---|---|
 
 
-# Shield Studies Addon Utils [![Build Status](https://travis-ci.org/tombell/travis-ci-status.svg?branch=master)](https://travis-ci.org/tombell/travis-ci-status)
+# Shield Studies Add-on Utils
+[![Build Status](https://travis-ci.org/mozilla/shield-studies-addon-utils.svg?branch=master)](https://travis-ci.org/mozilla/shield-studies-addon-utils)
 
-- Instrument your Firefox Addon!
-- Build Shield Study (Normandy) compatible addons without having to think very much.
+A Firefox javascript module to be bundled with shield study add-ons (as `StudyUtils.jsm`). Provides these capabilities:
 
-## Assumptions
+1. **Suggest variation for a client**
+2. **Report study lifecycle data** using Telemetry
+3. **Report feature interaction and success data** using Telemetry
+4. **Annotate Telemetry Environment** to mark the user as special in the `main` ping.
 
-- You are building a [LEGACY ADDONS](https://developer.mozilla.org/en-US/Add-ons/Legacy_add_ons).  To deploy these after 57, you will need the magic special signing.
--  Jetpack / addon-sdk is NOT SUPPORTED.
-- WebExtensions are not strong enough
+The pings end up in the `shield-study-addon` Telemetry bucket for faster analysis.
 
-## history of major versions
+Allows add-on developers to build [Shield Study][shield-wiki] (Normandy) compatible add-ons without having to think very much.
 
-- v4.x: (proposed)  additional functions for common cases
-- v4: First `.jsm` release.  Uses packet format for PACKET version 3.
-- v3: Attempt to formalize on `shield-study` PACKET version 3.  Jetpack based.  Prototype used for `raymak/page-reload`.  All work abandoned, and no formal npm release in this series.  Work done at `v3-shield-packet-format` branch.  LAST JETPACK (addon-sdk) RELEASE.
-s v2: Code refactor to es6 `class` with event models.  Added cli tooling.  Packet format is still arbitrary and per-study.  Jetpack based.  Last used in studies in Q2 2017.
-- v1: Initial work and thinking.  Telemetry packets are rather arbitrary.  Jetpack based.
+## What You are Building
 
+- You are building a [LEGACY ADD-ON](https://developer.mozilla.org/en-US/Add-ons/Legacy_add_ons). To deploy these after 57, you will need the magic special signing.
```

Comment:
nit: remove "/en-US/" from URL. MDN does a pretty awesome job of serving up the best available docs for users based on their preferred locale/language.